src/settings.rs

```diff
@@ -203,8 +203,7 @@ pub struct CommonSettings {
     ///     "triple": "x86_64-unknown-linux-gnu",
     ///     "is_ci": false,
     ///     "action": "Install",
-    ///     "status": "Failure",
-    ///     "failure_variant": "Symlink"
```

Comment:
I would be tempted to keep it in order to demonstrate what could be communicated in a failure condition, but since it is indeed not up-to-date, I don't feel strongly one way or the other (_could_ update it, but also could just accept this change).